packages/docusaurus-theme-classic/src/theme/NavbarItem/NavbarNavLink.tsx

```diff
@@ -33,32 +34,47 @@ export default function NavbarNavLink({
   const isExternalLink = label && href && !isInternalUrl(href);
   const isDropdownLink = activeClassName === dropdownLinkActiveClass;
 
+  // Link content is set through html XOR label
+  const linkContentProps = html
+    ? {dangerouslySetInnerHTML: {__html: html}}
+    : {
+        children: (
+          <>
+            {label}
+            {isExternalLink && (
+              <IconExternalLink
+                {...(isDropdownLink && {width: 12, height: 12})}
+              />
+            )}
+          </>
```

Comment:
This doesn't need to be a JSX fragment. Because `label` is a string, this can be simply an array without `key`.

Reply:
Anyways I guess it doesn't make the code any less verbose :(